### PR TITLE
Handle case if import with brackets is passed

### PR DESCRIPTION
**Pull request by alex-knyazev**

### CODE REVIEW

**Comment by the-stas** [comment]
@Semigradsky @alexgagnon can you please take a look?
thanks

File: lib/media-ast-from-string.js [hunk 1]
```diff
@@ -131,4 +132,11 @@ const andRegExp = new RegExp(`^${expressionRE}(?:${andRE}${spaceRE})$`, 'i');
 const spaceWrapRegExp = new RegExp(`^${noSpaceRE}${noExpressionRE}${noSpaceRE}$`);
 const mediaRegExp = new RegExp(`^(?:${modifierRE}${spaceRE})?(?:${typeRE}(?:${andRE}${spaceRE}${expressionRE})?|${expressionRE})$`, 'i');
 
+/**
+ * remove spaces after left and before right bracket if spaces exist
+ */
+function trimNearBrackets(string) {
+	return string.replace('( ', '(').replace(' )', ')')
```
Comment (Semigradsky) [inline lib/media-ast-from-string.js:139]: What if it will be two or more spaces?

Comment (the-stas) [reply lib/media-ast-from-string.js:139]: good point!
@alex-knyazev I guess we need trim all spaces

Comment (the-stas) [reply lib/media-ast-from-string.js]: @Semigradsky fixed, please check, thanks

**Comment by the-stas** [comment]
@jonathantneal can you please take a look?
thanks

**Comment by the-stas** [comment]
@Semigradsky can you please take a look?
thanks

**Comment by romainmenke** [comment]
Hi @alex-knyazev, @the-stas, @lipemat,

This plugin was moved to the csstools mono repo : https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-custom-media#readme

This PR was a great start but we wanted to handle a few more edge cases.
The fix has been released in v8.0.1

https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-custom-media/CHANGELOG.md#801-june-3-2022

**Pull request closed without merging**